gluster-block.spec.in

```diff
@@ -82,6 +82,7 @@ rm -rf ${RPM_BUILD_ROOT}
 %attr(0755,-,-) %{_initddir}/gluster-blockd
 %endif
 %config(noreplace) %{_sysconfdir}/sysconfig/gluster-blockd
+%config(noreplace) %{_sysconfdir}/logrotate.d/gluster-block
```

Comment:
You should mention about your change at %changelog section of this spec file.

Reply:
Okay, will do it.